src/test/java/io/appium/java_client/android/AndroidAppStringsTest.java

```diff
@@ -16,17 +16,19 @@
 
 package io.appium.java_client.android;
 
-import static org.junit.Assert.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 public class AndroidAppStringsTest extends BaseAndroidTest {
 
-    @Test public void getAppStrings() {
+    @Test
+    public void getAppStrings() {
```

Comment:
I guess such changes in the formatting can be avoided as well

Reply:
Resolved

Reply:
I see this particular formatting change and a lot of similar changes are still in place

Reply:
@valfirst I think i fixed all format issues manully(without any plugins) . If you see any other issues, please feel free to fix them.